game.js

```diff
@@ -1154,6 +1154,10 @@ class GameManager extends EventTarget {
   menuAim() {
     const localPlayer = metaversefileApi.useLocalPlayer();
     if (!localPlayer.hasAction('aim')) {
+      //current ads value is just a hardcoded offset value that seemed appropriate
+      cameraManager.saveaim();
```

Comment:
Is this missing from this PR?

Reply:
I actually removed it entirely in a more recent PR